### PR TITLE
Fix the "black screen of death"

### PR DESCRIPTION
**Pull request by openjck**

### CODE REVIEW

File: viewer/static/js/app/tests/jest/dataset-control-container.test.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+        B: { total_clients: 5000, total_pings: 5100 },
+        C: { total_clients: 6000, total_pings: 6100 },
+      },
+    },
+    {
+      date: '2017-07-31',
+      subgroups: ['Linux', 'All'],
+      populations: {
+        A: { total_pings: 600 },
+        B: {},
```
Comment (spasovski) [inline viewer/static/js/app/tests/jest/dataset-control-container.test.js]: good call

File: viewer/static/js/app/components/views/dataset-control.js [hunk 2]
```diff
@@ -37,10 +37,20 @@ export default function(props) {
...
-          const numClients = props.currentDataset.populations[cohort].total_clients.toLocaleString('en-US');
-          const numPings = props.currentDataset.populations[cohort].total_pings.toLocaleString('en-US');
+          const cohortMeta = props.currentDataset.populations[cohort];
+
+          let numClients = 0;
+          if (cohortMeta.total_clients) {
+            numClients = cohortMeta.total_clients.toLocaleString('en-US');
+          }
+
+          let numPings = 0;
```
Comment (spasovski) [inline viewer/static/js/app/components/views/dataset-control.js]: It seems if the API won't return anything we'll assume 0. Would a string saying "not provided" or something like it be better for the user?

Comment (openjck) [reply viewer/static/js/app/components/views/dataset-control.js]: Yeah good call

**Comment by openjck** [comment]
Updated! The counts will be omitted if 0.

**Comment by spasovski** [comment]
r+ thanks

**Pull request merged**